common/services/prismic/multi-content.js

```diff
@@ -6,6 +6,7 @@ import { parsePage } from './pages';
 import { parseEventSeries } from './events';
 import { parseBook } from './books';
 import { pagesFields } from './fetch-links';
+import { parseEventDoc as parseEvent } from '../../../server/services/prismic-parsers';
```

Comment:
This 100% won't work, so we'll have to refactor this first, which needed doing anyway